docs/source/basics/tokenizers.rst

```diff
@@ -0,0 +1,239 @@
+.. _tokenizers_usage_label:
+
+==========
+Tokenizers
+==========
+
+Tokenizers are a key component of any LLM. They convert raw text into token IDs, which index into embedding vectors that are
+understood by the model.
+
+In torchtune, tokenizers play the role of converting Messages object into token IDs and any necessary model-specific special tokens.
```

Comment:
Can you add a link to messages or give an example here? Otherwise it's not obvious what this means